enderio-machines/src/main/java/com/enderio/machines/data/datamap/RangeExtenderDataProvider.java

```diff
@@ -0,0 +1,36 @@
+package com.enderio.machines.data.datamap;
+
+import com.enderio.machines.common.datamap.RangeExtender;
+import com.enderio.machines.common.init.MachineBlocks;
+import com.enderio.machines.common.tag.MachineTags;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+import net.minecraft.core.HolderLookup;
+import net.minecraft.data.PackOutput;
+import net.minecraft.tags.TagKey;
+import net.minecraft.world.level.block.Block;
+import net.neoforged.neoforge.common.data.DataMapProvider;
+
+public class RangeExtenderDataProvider extends DataMapProvider {
+    private final Map<Block, Map<TagKey<Block>, Integer>> data = new HashMap<>();
+
+    public RangeExtenderDataProvider(PackOutput packOutput, CompletableFuture<HolderLookup.Provider> lookupProvider) {
+        super(packOutput, lookupProvider);
+    }
+
+    public void addData(TagKey<Block> tag, Block block, int value) {
+        data.computeIfAbsent(block, it -> new HashMap<>()).put(tag, value);
+    }
+
+    @Override
+    protected void gather() {
+        addData(MachineTags.Blocks.RANGE_EXTENDER, MachineBlocks.WIRELESS_CHARGER_ANTENNA.get(), 16);
```

Comment:
Are these figures a bit large?

Reply:
in what way?

Reply:
16 blocks per antenna would quickly outpace the size of many many chunks when surrounding the wireless charger, no? Unless it's just one antenna per machine in which case its okay.

Reply:
ah gotcha, thats fine then :)